### ISSUE TITLE
Not working on Android 11

### ISSUE THREAD

**Issue by ThePieMonster** [Mikotwa/FuckLocation#8]
Installed this module using LSPosed, checked only System Framework, and rebooted.

Opened up Google Maps and my location still remains the same.

I did enter in coordinates and clicked save, granted root access. (No negative coordinates can be entered though).

**Comment by Mikotwa** [comment]
Just upload new version 1.1.0, would you like to have a try and figure out if it works for you?

**Comment by ThePieMonster** [comment]
Regarding the below image, what is "x" and "y".
I assume "x" equals "Latitude". You should say "Latitude" instead of "x".

![image](https://user-images.githubusercontent.com/11359134/149681296-7385ed0f-2078-4844-a368-6874058ee044.png)
![image](https://user-images.githubusercontent.com/11359134/149681419-06fc75de-5086-4a1d-87dd-00ddc7f15d21.png)

------------------------------------------------------------------------------------------

Regarding the below image, what is "location whitelist"?
Can we get a search feature so we can type the app name in instead of scrolling?
When will "Test Settings" be completed"?

![image](https://user-images.githubusercontent.com/11359134/149681303-b1adc8cb-6626-400d-90ec-a706c85039b5.png)

------------------------------------------------------------------------------------------

@Mikotwa 
What application are you using to test? Should I see Google Maps display my current location as the location entered in the application?

**Comment by ThePieMonster** [comment]
Tested out v1.2.1 today @Mikotwa @zypA13510 

- I still suggest you replace the coordinated variable text with lat and lon instead of x and y.
- How are you testing your application? You should include testing instructions in the read me on github.
- Location whitelist setting needs a search and order feature
- Negative coordinates still do not work


![ezgif-7-1d27ec2d6b](https://user-images.githubusercontent.com/11359134/151419310-0d86fb9b-34d7-4b74-9e4c-d4a73928f7e8.gif)

**Comment by Mikotwa** [comment]
> I still suggest you replace the coordinated variable text with lat and lon instead of x and y.

Good idea. Will do that in the next (few) version.

> How are you testing your application? You should include testing instructions in the read me on github.

Basically, I'm using an Play application called Network Cell Info Lite. Testing instructions should coming out when Test Settings page is implemented.

> Location whitelist setting needs a search and order feature

Yeah, coming soon (Priority of UI is pretty low for now, so it takes longer time)

> Negative coordinates still do not work

v1.2.1.1 should works for now.

**Comment by Mikotwa** [comment]
v1.3.2.1 does some changes to improve experiences on Android 11. Would you like to have a try and see if it solves your problem? Thanks.

**Comment by ThePieMonster** [comment]
@Mikotwa 
v1.2.3.5 is the latest version I was able to download from the store.

App crashes when trying to save a negative decimal. 

Also want to point out that the version number should really only be 3 digits and not 4.
IE: v1.3.2 and not v1.3.2.1. There really isn't a need to have a fourth digit here.

**Comment by Mikotwa** [comment]
v1.2.4 merge some codes to crush this bug. Is it helps?

**Comment by ThePieMonster** [comment]
No. Still crashes. Have you tried testing this yourself before pushing code? @Mikotwa

**Comment by zypA13510** [comment]
> App crashes when trying to save a negative decimal.
v1.2.1.1 worked for me. In my case, I have to reboot after changing the coordinates. Also, I have disabled location accuracy (using Bluetooth and Wi-Fi to determine your location).

After upgrading, it starts crashing. I have to remove the entire config folder, rather than changing ownership as mentioned [here](https://github.com/Xposed-Modules-Repo/fuck.location/releases/tag/348-1.2.2). But the new version does not correctly mock location on my device. Guess that's a separate issue.

**Comment by ThePieMonster** [comment]
@zypA13510 Thanks for the info. I removed /data/system/fuck_location_test and uninstalled the app. Will come back and test once there is some more improvements as the app seems to have a lot of issues currently.
